config/importer.go

```diff
@@ -12,36 +12,31 @@ import (
 // Config struct to hold application configuration.
 type Config struct {
 	BindAddr                       string        `envconfig:"BIND_ADDR"`
+	ServiceAuthToken               string        `envconfig:"SERVICE_AUTH_TOKEN"                     json:"-"`
 	KafkaAddr                      []string      `envconfig:"KAFKA_ADDR"`
 	IncomingInstancesTopic         string        `envconfig:"DIMENSIONS_EXTRACTED_TOPIC"`
 	IncomingInstancesConsumerGroup string        `envconfig:"DIMENSIONS_EXTRACTED_CONSUMER_GROUP"`
 	OutgoingInstancesTopic         string        `envconfig:"DIMENSIONS_INSERTED_TOPIC"`
 	EventReporterTopic             string        `envconfig:"EVENT_REPORTER_TOPIC"`
 	DatasetAPIAddr                 string        `envconfig:"DATASET_API_ADDR"`
-	DatasetAPIAuthToken            string        `envconfig:"DATASET_API_AUTH_TOKEN"`
+	DatasetAPIAuthToken            string        `envconfig:"DATASET_API_AUTH_TOKEN"                 json:"-"`
 	DatabaseURL                    string        `envconfig:"DB_URL"`
```

Comment:
The database url shouldn't be logged as well, as it contains the user and password to neo4j